action.yml

```
@@ -20,6 +20,11 @@ inputs:
                 'major' - Assign to the next major version.
         required: false
         default: minor
+    target_milestone:
+        description: |
+            Milestone title for modifying any milestone.
```

This description is pretty hard to understand. Can you elaborate what `for modifying any milestone.` means?

---

If I'm understanding correctly, how about: 
```suggestion
            Title of the milestone that will be updated.
```

---

Yes, that's the perfect description. Thank you!